### PR TITLE
[CVP-1467]add support for the kube objects

### PR DESCRIPTION
**Pull request by jsztuka**

### CODE REVIEW

File: cmd/cvp-trigger/main.go [hunk 1]
```diff
@@ -105,6 +107,7 @@ func (o *options) gatherOptions() {
 	fs.StringVar(&o.operatorPackageName, operatorPackageNameOptions, "", "Operator package name to test")
 	fs.StringVar(&o.releaseImageRef, releaseImageRefOption, "", "Pull spec of a specific release payload image used for OCP deployment.")
 	fs.StringVar(&o.targetNamespaces, targetNamespacesOption, "", "A comma-separated list of namespaces the operator will target. If empty, all namespaces are targeted")
+	fs.StringVar(&o.pyxisUrl, PyxisUrl, "", "Represents cvp product package name for specific ISV")
```
Comment (nmars) [inline cmd/cvp-trigger/main.go:110]: Can we have same description as in the readme? `URL that contains specific cvp product package name for specific ISV with unique pid`

File: cmd/cvp-trigger/README.md [hunk 2]
```diff
@@ -72,6 +72,8 @@ CVP Trigger must load the configuration for the
    catalog will be installed. If empty, a new namespace will be created.
 - `OO_TARGET_NAMESPACES`: Optional. A comma-separated list of namespaces the
    operator will target. If empty, all namespaces will be targeted.
+- `PYXIS_URL`: Optional. URL that contains specific cvp product package name for specific ISV
+   with unique pid.
    If no OperatorGroup exists in `$OO_INSTALL_NAMESPACE`, a new one will be
```
Comment (nmars) [inline cmd/cvp-trigger/README.md:77]: Do lines 77-79 belong to `OO_TARGET_NAMESPACES` ?

File: cmd/cvp-trigger/main.go (hunk 1 above)
Comment (jsztuka) [reply cmd/cvp-trigger/main.go]: Sure thing, modified.

File: cmd/cvp-trigger/README.md (hunk 2 above)
Comment (jsztuka) [reply cmd/cvp-trigger/README.md]: I think so, changed order.

**Review by samvarankashyap — APPROVED** [review]
LGTM

**Review by nmars — APPROVED** [review]
lgtm

**Review by hongkailiu — COMMENTED** [review]
/lgtm

**Pull request merged**